ethereum/rpc/namespaces/eth/api.go

```diff
@@ -864,8 +864,45 @@ func (e *PublicAPI) GetTransactionReceipt(hash common.Hash) (map[string]interfac
 func (e *PublicAPI) PendingTransactions() ([]*rpctypes.RPCTransaction, error) {
 	e.logger.Debugln("eth_getPendingTransactions")
 
-	// FIXME https://github.com/tharsis/ethermint/issues/244
-	return []*rpctypes.RPCTransaction{}, nil
+	var result []*rpctypes.RPCTransaction
+	txs, err := e.backend.PendingTransactions()
+	if err != nil {
+		return nil, err
+	}
+	for _, tx := range txs {
+		if len((*tx).GetMsgs()) != 1 {
+			e.logger.Debugln("invalid tx")
+			return nil, fmt.Errorf("invalid tx type: %T", tx)
```

Comment:
this should `continue` (unless all txs are `MsgEthereumTx`)

Reply:
fixed